code/modules/antagonists/wizard/slaughter_antag.dm

```diff
@@ -50,20 +53,19 @@
 	if(objectives.len)
 		report += printobjectives(objectives)
 
-	var/datum/action/cooldown/spell/jaunt/bloodcrawl/slaughter_demon/consume_ability = locate() in owner.current?.actions
-	if(consume_ability?.consume_count > 0)
-		report += span_greentext("The [name] consumed a total of [consume_ability.consume_count] bodies!")
```

Comment:
Does `consume_count` do anything on `/datum/action/cooldown/spell/jaunt/bloodcrawl/slaughter_demon`? Should be removed if this was the only thing using it, otherwise keep it